src/test/java/org/ays/emergency_application/controller/EmergencyEvacuationApplicationEndToEndTest.java

```diff
@@ -320,7 +324,7 @@ void givenValidEmergencyEvacuationApplicationRequest_whenApplicationSaved_thenRe
         // Then
         String endpoint = BASE_PATH.concat("/emergency-evacuation-application");
         MockHttpServletRequestBuilder mockHttpServletRequestBuilder = AysMockMvcRequestBuilders
-                .post(endpoint, applicationRequest);
+                .post(endpoint, adminToken.getAccessToken(), applicationRequest);
```

Comment:
Public endpointlerde token göndermiyoruz
```suggestion
                .post(endpoint, applicationRequest);
```